charts/delphai-deployment/templates/scaled-object.yaml

```diff
@@ -0,0 +1,22 @@
+{{ if $.Values.rabbitmq.enabled }}
+
+{{ $rabbitmqSecret := (lookup "v1" "Secret" .Release.Namespace "rabbitmq") }}
+
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  namespace: {{ $.Release.Namespace | quote }}
+  name: {{ $.Release.Name | quote }}
+spec:
+  minReplicaCount: 1
+  scaleTargetRef:
+    name: {{ $.Release.Name | quote }}
+  triggers:
+  - type: rabbitmq
+    metadata:
+      host: {{ index $rabbitmqSecret.data "connection-string" | b64dec }}
```

Comment:
That fetches and hardcodes the secret in clear text here, not good

You can link a secret here https://keda.sh/docs/2.12/concepts/authentication/#secrets
Or get it from an env var https://keda.sh/docs/2.12/concepts/authentication/#environment-variables

Reply:
The RabbitMQ secret got encapsulated in a `TriggerAuthentication` object in 777d0a0441c1efca015b17d3c1fefc7959a577a4